RELEASES.md

```diff
@@ -1,25 +1,30 @@
 # Release Notes
 
-## 0.11.0 (Unreleased)
+## 0.11.0 (4-28-2015)
 - Export filesystem usage metrics for the node and containers.
+- Support for Kubernetes events
+- New Sink - Google Cloud Logging. Supports events only.
+- New Sink - Google Cloud Monitoring. Supports metrics only.
+- New metric labels - 'pod_name' and 'resource_id'.
+- Extensible source and sinks configuration. It is now possible to export to multiple sources and sinks.
```

Comment:
I think you mean it is now possible to have multiple sources and sinks. We don't export to sources :)